src/Hl7.Fhir.Support/Hl7.Fhir.Support.csproj

```diff
@@ -15,7 +15,7 @@
 		<AssemblyName>Hl7.Fhir.Support</AssemblyName>
 	</PropertyGroup>
 
-	<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' or '$(TargetFramework)' == 'net5.0'">
```

Comment:
I don't think this library is required by net 45, so probably better to change it to:
`'$(TargetFramework)' == 'netstandard1.6' or $(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'net5.0'`

Reply:
Or `'$(TargetFramework)' != 'net45`